templates/mcollective/mcollective-client.cfg.erb

```diff
@@ -3,8 +3,15 @@
 main_collective = mcollective
 collectives = mcollective
 libdir = <%= scope.lookupvar('::openshift_origin::params::ruby_scl_path_prefix') %>/usr/libexec/mcollective
+<% if scope.lookupvar('::openshift_origin::ose_version') != "" -%>
+# logger_type = file cannot be set for OpenShift Enterprise
+# # https://bugzilla.redhat.com/show_bug.cgi?id=963332
+logger_type = console
+loglevel = warn
+<% else -%>
```

Comment:
So, this works in origin but not ose? That seems odd as I doubt we're customizing mcollective. John is assigned the referenced bug, lets see what he says, ping @dobbymoodge